backend/coreapp/migrations/0021_rename_assembly_asm.py

```diff
@@ -4,6 +4,7 @@
 
 
 class Migration(migrations.Migration):
+    atomic = False
```

Comment:
This is maybe a tiny bit less safe than it could be on Postgres, but this migration can't run against a sqlite DB without it? And I don't think it'll be an issue moving forward?

Reply:
yeah, no problem. I think we'll compress all the migrations soon anyway